patches/tModLoader/Terraria/DataStructures/GameModeData.TML.cs

```diff
@@ -0,0 +1,72 @@
+using Terraria.GameContent.Creative;
+
+namespace Terraria.DataStructures
+{
+	public partial class GameModeData
+	{
+		public float GetPureEnemyDamageModifier() {
+			float enemyDamageModifier = EnemyDamageMultiplier;
+
+			if(IsJourneyMode) {
+				CreativePowers.DifficultySliderPower difficultySliderPower =
+					CreativePowerManager.Instance.GetPower<CreativePowers.DifficultySliderPower>();
+				enemyDamageModifier *= difficultySliderPower.StrengthMultiplierToGiveNPCs;
+			}
+			
+			return enemyDamageModifier;
+		}
+
+		public float GetEnemyDamageModifier() {
+			float enemyDamageModifier = GetPureEnemyDamageModifier();
+			if (Main.getGoodWorld) {
+				if (IsJourneyMode) {
+					enemyDamageModifier *= 2;
+				}
+				else {
+					enemyDamageModifier += 1;
```

Comment:
Vanilla code for this is just ...
I think this should be `enemyDamageModifier = enemyDamageModifier + 1) * (1 + 1/3f);`

because `ScaleStats_ApplyGameMode` has
```
			if (Main.getGoodWorld)
				damage += damage / 3;
```
at the end